documents/src/pages/utils/navigation-grid.md

```diff
@@ -35,180 +35,211 @@ cell = down(grid, cell); // => [2, 1], third column and second row
 
 A common use case for the utility is to provide navigation over the HTML table.
 
-::
-```css
-.table {
-  display: table;
-  border-spacing: 1px;
-}
-
-.row {
-  display: table-row;
-}
-
-.cell {
-  font-size: 16px;
-  display: table-cell;
-  width: 3em;
-  height: 3em;
-  text-align: center;
-  vertical-align: middle;
-  outline: none;
-  background-color: #A01C2B;
-  cursor: pointer;
-}
-
-.cell[tabindex] {
-  background-color: #227542;
-}
+Click on the table and use _Arrow_, _Home_ and _End_ keys to select cells.
 
-.cell:focus {
-  background-color: #334BFF;
-}
-```
+::
 ```html
-<div id="grid" class="table">
-  <div class="row">
-    <div class="cell" tabindex="0">1</div>
-    <div class="cell">0</div>
-    <div class="cell" tabindex="0">1</div>
-    <div class="cell">0</div>
-  </div>
-  <div class="row">
-    <div class="cell" tabindex="0">1</div>
-    <div class="cell" tabindex="0">1</div>
-    <div class="cell" tabindex="0">1</div>
-    <div class="cell" tabindex="0">1</div>
-  </div>
-  <div class="row">
-    <div class="cell" tabindex="0">1</div>
-    <div class="cell" tabindex="0">1</div>
-    <div class="cell">0</div>
-    <div class="cell" tabindex="0">1</div>
-  </div>
-  <div class="row">
-    <div class="cell">0</div>
-    <div class="cell" tabindex="0">1</div>
-    <div class="cell" tabindex="0">1</div>
-    <div class="cell" tabindex="0">1</div>
-  </div>
-</div>
+<efx-navigation-grid></efx-navigation-grid>
 ```
+
 ```javascript
 import { halo } from '/theme-loader.js';
+import { BasicElement, html, css } from 'https://cdn.skypack.dev/@refinitiv-ui/core?min';
+import { customElement } from 'https://cdn.skypack.dev/@refinitiv-ui/core/decorators/custom-element.js?min';
+import { ifDefined } from 'https://cdn.skypack.dev/@refinitiv-ui/core/directives/if-defined.js?min';
 import { first, last, left, right, up, down } from 'https://cdn.skypack.dev/@refinitiv-ui/utils/navigation.js?min';
 halo();
 
-const gridElement = document.getElementById('grid');
+// Number of rows and columns to generate random sample matrix
+const Rows = 4;
+const Columns = 4;
+const Matrix = Array.from(Array(Rows)).map(
```

Comment:
It's always generate a random sample matrix when I opened and closed `code-sandbox`

Reply:
Well, yeah, it regenerates the code :) I think it is a bit more fun when on page loads it gives different result